zello-sdk-sample/src/main/java/com/zello/sdk/sample/AnotherActivity.java

```diff
@@ -265,7 +266,8 @@ private void updateMessageState() {
 		} else if (incoming) {
 			String author = _messageIn.getAuthor().getDisplayName(); // Is message from channel?
 			if (author != null && author.length() > 0) {
-				_txtMessageName.setText(_messageIn.getFrom().getDisplayName() + " \\ " + author); // Show channel and author names
+				// Show channel and author names
+				_txtMessageName.setText(String.format(getResources().getString(R.string.message_in_from_channel), _messageIn.getFrom().getDisplayName(), author));
```

Comment:
You should be able to omit the getResources() call and just call getString() directly from the Activity context